service/lib/agama/dbus/manager.rb

```diff
@@ -65,7 +65,7 @@ def initialize(backend, logger)
         dbus_method(:Commit, "") { install_phase }
         dbus_method(:CanInstall, "out result:b") { can_install? }
         dbus_method(:CollectLogs, "out tarball_filesystem_path:s") { collect_logs }
-        dbus_method(:Finish, "") { finish_phase }
+        dbus_method(:Finish, "in method:s, out result:b") { |m| finish_phase(m) }
```

Comment:
You should update the [D-Bus documentation](https://github.com/agama-project/agama/tree/master/doc/dbus), too. However, I am fine with you merging this PR and updating the docs later.

Remember: if you update a proxy, you should update the docs too.

PS: I do not think that documentation is that useful.